src/main/java/de/cronn/diff/html/DirectoryDiffHtmlBuilder.java

```diff
@@ -62,29 +62,29 @@
 
 	private static final String CSS_CLASS_ONE_LINE_HEADER = "one-line-header";
 
-	private String title;
+	private final String title;
```

Comment:
I don't agree with adding these `final` keywords in this class. We might even want to re-assign one of these class members at some point, plus it blows up the code without need.